budget/tests.py

```diff
@@ -498,7 +498,8 @@ def test_rounding(self):
         result[models.Project.query.get("raclette").members[0].id] = 8.12
         result[models.Project.query.get("raclette").members[1].id] = 0.0
         result[models.Project.query.get("raclette").members[2].id] = -8.12
-        self.assertDictEqual(balance, result)
+        for key, value in balance.iteritems():
```

Comment:
Maybe just add a comment to explain why you have to do that weird (but required) thing.